src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -71,7 +71,7 @@ static License GetTrialLicense()
             }
             else
             {
-                var message = $"Trial for Particular Service Platform is still active, trial expires on {trialLicense.ExpirationDate.Value.ToLocalTime().ToShortDateString()}.";
+                var message = $"Trial for the Particular Service Platform has been active since {trialStartDate.ToLocalTime().ToShortDateString()}.";
```

Comment:
Whilst the rest of your suggested message mitigates that feeling, we felt that this isn't really necessary in the log message, since a) this shouldn't be running in any kind of unattended manner in a way people are relying on it (it's just  a trial) and b) the dialogs take care of the actual call to action, since the main activity during the trial will be running apps interactively.